beers/models.py

```diff
@@ -0,0 +1,62 @@
+import string
+
+from django.db import models
+
+class BeerStyleCategory(models.Model):
+    CLASS_CHOICES = (
+        ('beer', 'Beer'),
+        ('cider', 'Cider'),
+        ('mead', 'Mead'),
+    )
+
+    name = models.CharField(max_length=100, unique=True, db_index=True)
+    bjcp_class = models.CharField(max_length=10, choices=CLASS_CHOICES, default='beer')
+    notes = models.TextField(blank=True)
+    category_id = models.PositiveSmallIntegerField()
```

Comment:
Make category + revision + class unique and that'll prevent the incorrect import from causing duplicate category IDs.